lrauv_ignition_plugins/test/helper/LrauvTestFixture.hh

```diff
@@ -171,10 +171,32 @@ class LrauvTestFixture : public ::testing::Test
     return pid;
   }
 
+  public: static void ExportLogs(std::string _target)
+  {
+    int res = system("sudo /home/developer/lrauv_ws/src/lrauv/lrauv_ignition_plugins/plots/unserialize_for_plotting.sh");
+    
+    if (res != 0)
+    {
+      ignerr << "Failed to unserialize plots\n";
+      return;
+    }
+    ignmsg << "Finished unserialize, copying.\n";
+    
+    // WARNING: THIS CAN LEAD TO ARBITRARY CODE EXECUTION
+    auto targetfile = _target + ".csv";
+    auto cmd = std::string("sudo cp -r /home/developer/lrauv_ws/src/lrauv/lrauv_ignition_plugins/plots/missions/tmp/tmp.csv /results/") + targetfile;
```

Comment:
Is there a way we can configure it not to do this when running the tests locally? These files can quickly start to take up space, and since they're ending up far from the code, they may be hard to track down.

Maybe we can do `export STORE_LOGS=true` on CI?